src/app/[locale]/(business)/(with-header)/users/[username]/cellar/_components/cellar-filters/index.tsx

```
@@ -0,0 +1,113 @@
+"use client";
+
+import { ServingFrom } from "@db/enums";
+
+import { useRouter, useSearchParams } from "next/navigation";
```

ESLint

[error] 'useRouter' import from 'next/navigation' is restricted. Please import navigation utilities from "@/lib/i18n" instead.

(no-restricted-imports)